docs/pages/Troubleshooting.md

```diff
@@ -12,7 +12,7 @@ Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/b
 ## UI5 Project
 ### Corrupt Framework Dependencies
 
-When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers like npm or Yarn.
+When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers e.g. npm.
```

Comment:
I wonder if this should be kept in the 3.0 documentation. it makes sense for v2, but not for v3, isn't it?

Reply:
Thats true. I have just went over the yarn occurrences not checking the whole section. I'll remove the whole section

Reply:
done